tidy3d/components/simulation.py

```diff
@@ -187,17 +187,6 @@ class Simulation(Box):  # pylint:disable=too-many-public-methods
 
     """ Validating setup """
 
-    @pydantic.validator("symmetry", always=True)
-    def not_supported_yet(cls, val):
```

Comment:
so as of this PR symmetry is only supported for mode computation?  Or am I missing something?

Reply:
correct

Reply:
Then we should edit the docstring for Simulation.symmetry temporarily?

Reply:
Well I am still hoping to get the main symmetries done in the next couple days... Sooo no need?